extensions/modules/src/org/exist/xquery/modules/xslfo/ApacheFopProcessorAdapter.java

```diff
@@ -126,7 +126,9 @@ private FOUserAgent setupFOUserAgent(final FOUserAgent foUserAgent, final Proper
      */
     private ResourceResolver getResourceResolver(final DBBroker broker, final String baseUri) {
         final ResourceResolverFactory.SchemeAwareResourceResolverBuilder builder = ResourceResolverFactory.createSchemeAwareResourceResolverBuilder(ResourceResolverFactory.createDefaultResourceResolver());
-        final URIResolverAdapter uriResolver = new URIResolverAdapter(new ExistSchemeRewriter(new EXistURIResolver(broker, baseUri)));
+        final URIResolverAdapter uriResolver = new URIResolverAdapter(
+            new ExistSchemeRewriter(new EXistURIResolver(broker.getBrokerPool(), baseUri))
+        );
         builder.registerResourceResolverForScheme("exist", uriResolver);
```

Comment:
I did not say that we must not support `xmldb:exist:/` but that we must start to support `xmldb:/`